Installer.php

```diff
@@ -132,46 +132,49 @@ protected function addPackage(PackageInterface $package)
         $this->saveExtensions($extensions);
     }
 
-    protected function generateDefaultAlias(PackageInterface $package)
```

Comment:
Why are you removing protected method? This is BC break.

Reply:
@bizley change name to correct, it's protected method + I'm don't find any packages extends that `Installer`

Reply:
It doesn't matter that there are no public packages that extend it. Only private methods can be changed freely like that.